### PR TITLE
scripts added

### PR DESCRIPTION
**Pull request by amanhans306**
added new python scripts

### CODE REVIEW

**Comment by RIGUPTA2** [comment]
In 2_...script
creation date
C capital
D capital
=============
How to use numbers in python as a calculator
Improve this statement
=============
Intialize variables  
+# Perfom arthmetic operation by using that variables 

Perfom arthmetic operation by using initialized variables
=============
+# How to use numbers in python as a calculator

Perform basic arithmetic operations in python like addition, subtraction, multiplication, divison, log, etc...
=============
+# Grab everything, but go in step sizes of 2 from "abcedefghijk"

This problem statement needs elaboration
=============
+# print a string backwards ; eg: "helloworld"
P capital
=============
+# change the first letter of  word "Sam " to "R", so as to make it "Ram"
C capital
=============
+# Use built in function to Upper case ,Lower case and Split the string"Hello World" 
Grammatically incorrect
=============

**Comment by RIGUPTA2** [comment]
Closing it. You need to reopen after changes.

File: PythonScripts/1_CommandLineBasic [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# How to work command line  
+
+# Author Name
+# Creation Date 
+# Purpose
+
+
+# Add command for Finding Currnet Directory , Change Directory , How to clear command line screen
```
Comment (RIGUPTA2) [inline PythonScripts/1_CommandLineBasic:8]: Remove space before comma

Comment (amanhans306) [reply PythonScripts/1_CommandLineBasic:8]: noted sir

File: PythonScripts/1_CommandLineBasic [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# How to work command line  
+
+# Author Name
+# Creation Date 
+# Purpose
+
```
Comment (RIGUPTA2) [inline PythonScripts/1_CommandLineBasic:6]: Remove this line

**Pull request closed without merging**